packages/excalidraw/components/ConfirmDialog.tsx

```diff
@@ -43,7 +44,10 @@ const ConfirmDialog = (props: Props) => {
           onClick={() => {
             setAppState({ openMenu: null });
             setIsLibraryMenuOpen(false);
-            onCancel();
+            flushSync(() => {
+              onCancel();
+            });
+
```

Comment:
if this is also needed, then let's also comment about the `131.0.6778.x` bug for posterity

Reply:
does not crash now, but I would rather keep it safe, as it depends on the `onCancel` prop which is defined above